dokku.go

```diff
@@ -156,3 +160,25 @@ func (d *Dokku) urls(appName string) ([]string, error) {
 	}
 	return result, nil
 }
+
+// cleanupDeadContainers removes any container with the "Dead" status returned by API.
+// Such containers are commonly left behind by 'dokku cleanup'
+func (d *Dokku) cleanupDeadContainers() error {
+	listOpts := docker.ListContainersOptions{All: true}
+	list, err := d.Client.ListContainers(listOpts)
+	if err != nil {
+		return err
+	}
+
+	for _, container := range list {
+		if container.Status == "Dead" {
```

Comment:
Status returned to client by API is the same as you see by `docker ps`, so e.g. `Dead`, `Exited (137) 15 hours ago` etc.